onnx/defs/schema.h

```diff
@@ -1060,6 +1061,7 @@ class OpSchemaRegistry final : public ISchemaRegistry {
 
   static const std::vector<OpSchema> get_all_schemas() {
     std::vector<OpSchema> r;
+    r.reserve(4);
```

Comment:
We do not want hardcoded reserve(). 
In general, while reserve() usage may be beneficial, it will not make most of the code vastly more efficient. There are two reasons for that 1) the types we often use are primitive types, cheap to copy 2) in addition, the amount of elements is very small, it would fit into a single allocation block if such allocation is necessary at all, because std::vector as well std::string now has some inline space, which eliminates a need for any allocations/re-allocations for many cases.

Reply:
As I know std::vector implementations does not have SVO/SBO. It's explained here: https://stackoverflow.com/questions/48669413/is-sso-used-in-any-other-standard-library-containers-other-than-stdstring
But I will remove that for the same reason as before: it's hard-coded number.